### ISSUE TITLE
Carousel - Keyboard navigation doesn't work

### ISSUE THREAD

**Issue by mhilland** [OKAMca/stack#342]
On the Carousel component, the left/right arrows should be able to be triggered with the keyboard enter/space keys
